media/js/firefox/new/all-test-variations-pt2.js

```diff
@@ -0,0 +1,127 @@
+            // pull the nojs links out of the primary CTA
+            var $directLis = $dlButton.find('.nojs-download li').remove();
+            // container to hold direct download links
+            var $modalDirectDownloadList = $('#test-direct-downloads');
+            // os's to filter out of modal (as we have the app store specific buttons displayed already)
+            var mobileOs = ['android', 'ios'];
+            // array to hold re-sorted download li's
+            var sortedLis = [];
+
+            // remove button-y CSS
+            $directLis.find('a').removeClass('button green');
+
+            // massage the download links
+            $directLis.each(function(i, li) {
+                // do not include mobileOs's
+                if ($.inArray($(li).find('a').data('download-os').toLowerCase(), mobileOs) === -1) {
+                    sortedLis.push(prepLi(li));
+                }
+            });
+
+            // sort the download links
+            sortedLis.sort(function(a, b) {
+                if (Number($(a).attr('data-sort')) < Number($(b).attr('data-sort'))) {
+                    return 1;
+                } else {
+                    return -1;
+                }
+            });
+
+            // add the download links to the DOM
+            for (var i = sortedLis.length; i >= 0; i--) {
+                $modalDirectDownloadList.append(sortedLis[i]);
+            }
+
+            // set up modal-opening link underneath primary CTA for version 1
+            if (version === 1) {
+                // get current user platform display name
+                var platformDisplayName = $dlButton.find('.download-list li:visible a').data('display-name');
+
+                if (platformDisplayName) {
+                    // put "for {user's os}" text underneath modal primary dl button
+                    $('#test-modal-user-platform').text('for ' + platformDisplayName);
+                } else {
+                    $('#test-modal-user-platform').remove();
+                }
+
+                // conjure up a new link that will trigger the modal
+                var $newLink = $('<button id="test-modal-link">Download Firefox for another platform</button>');
+
+                $newLink.on('click', function(e) {
+                    e.preventDefault();
+
+                    // open up said modal
+                    Mozilla.Modal.createModal(this, $('#test-modal'));
+
+                    dataLayer.push({
+                        'event': 'alternate-version',
+                        'link-name': 'Systems & Languages'
+                    });
+                });
+
+                // place the new link underneath the main download button
+                $dlButton.append($newLink);
+            // move forest up on the page a bit (original value = -254px)
+            } else if (version === 2) {
+                $('.forest-container').css('margin-top', '-310px');
```

Comment:
Moving this content up is fine as long as the download buttons are shown, but if you set the HTML classes `linux arm` for example, then the install link can no longer be clicked because the horizon assets cover the content.